src/lando/main/models/commit_map.py

```diff
@@ -0,0 +1,90 @@
+import requests
+from django.db import models
+
+from lando.main.models import BaseModel
+from lando.main.models.repo import Repo
+
+
+class CommitMap(BaseModel):
+    """Map a git hash to an hg hash, based on a specific repo."""
+
+    HGMO_PUSHLOG_TEMPLATE = "https://hg.mozilla.org/{}/json-pushes"
+    REPO_MAPPING = (("firefox", "mozilla-unified"),)
+
+    git_hash = models.CharField(default="", unique=True)
+    hg_hash = models.CharField(default="", unique=True)
+
+    # NOTE: This value is set because multiple Lando repos can map to a single hg repo.
+    # This is because currently a separate repo object is created for each branch of a
+    # git repo, though those might map to a single hg repo (e.g., mozilla-unified).
+    git_repo_name = models.CharField(default="")
```

Comment:
Does this mean we may have the same hash mapping multiple times (once per repo)?

(My previous comment was just about preventing a serendipitous hash collision, but this may precipitate the issue.)

Reply:
It is possible but unlikely (mainly if the two repos are for different products, i.e., different repos on hgmo).